### PR TITLE
chore: setup PHP Coding Standards

### PR DESCRIPTION
**Pull request by arkaitzgarro**
Adhere to [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) and [Symfony](https://symfony.com/doc/current/contributing/code/standards.html) coding standards.

### CODE REVIEW

File: src/Agent.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function __construct(array $config, float $request_start_time)
 
     public function registerCollectors(Application $app): void
     {
-        if (config('elastic-apm-laravel.spans.querylog.enabled') !== false) {
+        if (false !== config('elastic-apm-laravel.spans.querylog.enabled')) {
```
Comment (countless-integers) [inline src/Agent.php:40]: yoda conditionals making a come-back I see...

File: src/Collectors/DBQueryCollector.php [hunk 2]
```diff
@@ -84,7 +80,7 @@ private function getQueryName(string $sql): string
 
             if (isset($query_type) && is_array($tables)) {
                 // Query type and tables
-                return $query_type . ' ' . join(', ', array_values($tables));
+                return $query_type.' '.join(', ', array_values($tables));
```
Comment (countless-integers) [inline src/Collectors/DBQueryCollector.php]: Is this really in the coding standard? Readability is horrible...

**Review by countless-integers — COMMENTED** [review]
not a big fan of some of the rules. Merge at your own peril.

File: src/Collectors/DBQueryCollector.php (hunk 2 above)
Comment (arkaitzgarro) [reply src/Collectors/DBQueryCollector.php]: Yeah, I also think this is not good for readability. I'm going to tweak some rules.

**Comment by arkaitzgarro** [comment]
I'm merging it, and might open a separate PR to fix variable and configuration cases.

**Pull request merged**